src/main/java/entities/User.java

```diff
@@ -1,7 +1,7 @@
-package entities;
-
-public class User extends AbstractUser{
```

Comment:
It says that AbstractUser has been deleted, so User can't inherit from AbstractUser. Is this a mistake on my page?

Reply:
Talk to @Shaffaan about it but apparently, we no longer use the AbstractUser class. Let me know if I should be keeping it though.

Reply:
@fishyvishy I changed abstract user, and we do not have it anymore. The new classes are going to be a user interface and a common user which implements user interface. I will be sending a pull req for those really soon.